requirements.txt

```diff
@@ -12,6 +12,7 @@ solrq
 validators
 keyring;platform_system=='Darwin'
 boto3
+deprecation
```

Comment:
I don't see where this is being used. We need to be careful adding too many new packages.